### PR TITLE
New feature - flactag

### PR DESCRIPTION
**Pull request by aszweryn**
A tool to add or change metadata. Waiting for feedback.

### CODE REVIEW

File: audiotoolbox.sh [hunk 1]
```diff
@@ -43,3 +43,33 @@ flacer(){
 	echo -e "\e[1;33mConversion complete: "${1%.*}.flac" in $PWD\e[0m"
 	fi
 }
+
+flactag(){
+	filePath="${1:-Unknown}"
+	if test -f "$filePath"; then
+	picturePath="${2:-Unknown}"
+	echo -e "\e[1;33mRemoving old metadata...\e[0m"
+	metaflac --remove-all "$filePath"
```
Comment (Arthur-Sebastian) [inline audiotoolbox.sh:52]: Removing metadata from a file without prompting the user first. Recommended to implement a check for existing metadata, display it and then ask the user if they want to remove/keep/edit the data.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,28 @@ Note: Argument [quality] has a default value of 0 which corresponds to
...
+### flacer
+A script converting various audio files into a standard 16bit/44.1kHz flac format.<br>
+Usage:
+> flacer [filepath]
+
+Arguments:
+- filepath: path to the audio file to be converted
+
+### flactag
+Tool for metadata addition or editoin of flac filesm, automatically naming the file:
```
Comment (Arthur-Sebastian) [inline README.md:71]: typos: metadata edition*, files*;
Naming the file is not a part of the script's functionality - it has the form of a tagging wizard. Instructions need to reflect that somehow.

File: README.md [hunk 3]
```diff
@@ -59,6 +59,28 @@ Note: Argument [quality] has a default value of 0 which corresponds to
...
+>[Artist] - [Title]<br>
+
+Usage:
+> flactag [filepath of flac] [filepath of image]
```
Comment (Arthur-Sebastian) [inline README.md:75]: Argument names have to be short and descriptive. Recommended substitutes: file; image.

File: README.md [hunk 4]
```diff
@@ -59,6 +59,28 @@ Note: Argument [quality] has a default value of 0 which corresponds to
...
+
+Arguments:
+- filepath of flac: location of the audio file
+- filepath of image: location of the image that will be set as an album conver. It can be both of jpg and png formats and have any aspect ratio.
```
Comment (Arthur-Sebastian) [inline README.md:79]: Argument description should provide all the necessary information, instead of trying to contain it in argument's name.

**Review by Arthur-Sebastian — CHANGES_REQUESTED** [review]
README contains some typos, misleading information. Script also requires a minor tweak.

File: audiotoolbox.sh (hunk 1 above)
Comment (aszweryn) [reply audiotoolbox.sh:52]: Will do

File: README.md (hunk 2 above)
Comment (aszweryn) [reply README.md:71]: It actually do change the name of the file, check line 63 in audiotoolbox.sh

**Pull request closed without merging**